python/cudf/cudf/_lib/cpp/io/arrow_io_source.pxd

```diff
@@ -11,5 +11,5 @@ cdef extern from "cudf/io/arrow_io_source.hpp" \
         namespace "cudf::io" nogil:
 
     cdef cppclass arrow_io_source(cudf_io_datasource.datasource):
-        arrow_io_source(string arrow_uri) except +
+        arrow_io_source(const string& arrow_uri) except +
```

Comment:
east const.

Reply:
For some reason I only see "const string" in pxd files so I kept this consistent with other instances. I'm not sure if our east const guideline reached Cython.

Reply:
Cython doesn't support east const.